sched/Kconfig

```diff
@@ -990,6 +990,14 @@ config SCHED_INSTRUMENTATION_IRQHANDLER
 
 			void sched_note_irqhandler(int irq, FAR void *handler, bool enter);
 
+config SCHED_INSTRUMENTATION_HIRES
+	bool "Use Hi-Res RTC for instrumentation"
+	depends on RTC_HIRES
```

Comment:
should we decopule SCHED_INSTRUMENTATION_HIRES from RTC_HIRES? since many system timer support the microsecond now, on the other hand, ony few RTC support the high resolution.